### DISCUSSION TITLE
最近折腾的几个小特性

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#34] [General]
# 最近折腾的几个小特性

最新开发的两个功能特性：Obsidian插件personal assistant通过Chartjs Animation支持统计数据渐进式动画展示，使用户界面更为精细和连贯；松烟阁与Memos通过ClustrMap添加域名和Ghost Code动态注入引入用户足迹的动、静态展示

https://www.edony.ink/new-awesome-features-to-show-off/

**Comment by ghost** [comment]
不知道是不是我的问题，你的域名被重置了，是否被墙了。

**Reply by edonyzpc** [reply]
是不是因为被误识别成机器人了，你的 HTTP Request 能发一下看看吗？
<img width="798" alt="image" src="https://github.com/edony-ink/comments/assets/7744664/992fcb73-0489-4467-b4ed-54a26c375d23">

**Reply by ghost** [reply]
这个怎么看，在哪里看啊？😅

**Reply by edonyzpc** [reply]
emm，如果你不熟悉这些的话，可以告诉我你是浏览器访问还是其他什么应用，以及你的访问方式？我去测试看看。

***P.S.*** 如果浏览器访问 `http://edony.ink/rss` 能够返回 xml 文件的话（如下图所示），那么大概率是被识别成 bot 了，所以需要确认你的 HTTP request 加一下白名单
<img width="873" alt="image" src="https://github.com/edony-ink/comments/assets/7744664/39d58b5a-6c12-43dd-b689-18a5dad02abd">

**Reply by ghost** [reply]
之前用ssrp+的时候就突然有天访问不了我就加入了代理列表，目前没问题了，也不知道是不是分流的问题还是怎么滴。我换了openclash就ok了。
![QQ截图20240516110904](https://github.com/edony-ink/comments/assets/98329765/70e1be5d-8d53-4085-af6a-9615df6d8968)
